apps/bestofjs-nextjs/src/components/search-palette/search-palette.tsx

```diff
@@ -149,6 +149,10 @@ export function SearchPalette({ allProjects, allTags }: SearchProps) {
     });
   };
 
+  const isEmptyProjects = filteredProjects.length == 0,
+    isEmptyTags = filteredTags.length == 0,
+    isEmptySearchResults = isEmptyProjects && isEmptyTags;
+
```

Comment:
Even if it's standard JS, I would avoid the use of the `,` to separate variable declarations, could you use boring `const` for each line, to make things explicit and for consistency with the rest of the codebase?

Reply:
Updated thanks.